### DISCUSSION TITLE
where to find Free GPUs(A100, H100) ?

### DISCUSSION THREAD

**Discussion by aipdv** [community/community#138822] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

ive been working onto a project related to "sakana ai scientist" i was trying to integrate the opensourced llms onto it, but the project itself requires highend GPUs(greater than sm_80(sm means streaming multiprocessor)), like A100 and H100. So anyone has any knowledge if there are some platforms available which provides it? for Free??

**Comment by typed-sigterm** [comment]
Unfortunately there is no free hardware, just like there is no free lunch

**Comment by LiteBrite82** [comment]
Hi @xprabhudayal ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
